src/CLI/clitree/cli-xml/ts.xml

```diff
@@ -125,7 +218,19 @@
                    >
               </PARAM>
 
-	   <ACTION builtin="clish_pyobj">sonic_cli_tam_int_ifa_ts patch_sonic_tam_int_ifa_ts_sonic_tam_int_ifa_ts_tam_int_ifa_ts_flow_table_tam_int_ifa_ts_flow_table_list ${flow-name} ${acl-table-name} ${acl-rule-name}</ACTION>
+           <ACTION builtin="clish_pyobj">sonic_cli_tam_int_ifa_ts patch_sonic_tam_int_ifa_ts_sonic_tam_int_ifa_ts_tam_int_ifa_ts_flow_table_tam_int_ifa_ts_flow_table_list ${flow-name} ${acl-table-name} ${acl-rule-name}</ACTION>
+           <DOCGEN>
+                <DESCRIPTION>
+                   Configure Tail Timestamping flow parameters.
+                </DESCRIPTION>
+                <USAGE>
+                   Use this command to configure Tail Timestamping flow table.
+                </USAGE>
+                <EXAMPLE summary="Tail Timestamping flow configuration">
+                   dhcp-10-59-142-164(config-int-ifa-ts)# flow flow32 acl-table tacl1 acl-rule trule1
+                   dhcp-10-59-142-164(config-int-ifa-ts)#
```

Comment:
We can do away with redundant empty prompt. to keep it clean

Reply:
Empty prompt kept to show that no message comes after running this command.